get.sh

```diff
@@ -160,6 +160,33 @@ parseCommandLineArgs()
 	echo "TESTDIR: $TESTDIR"
 }
 
+# If the current directory contains only a single directory then squash that directory into the current directory
+squashSingleFolderContentsToCurrentDir()
```

Comment:
We discussed this problem with z/OS earlier, but what was the problem that this function solved?

Reply:
it removes a folder if the archive has just a single top level folder

Reply:
should I call it "stripFolderIfArchiveContainsOnlyOne" ?

Reply:
> should I call it "stripFolderIfArchiveContainsOnlyOne" ?

Up to you on the name - I don't mind too much.

Reply:
i'll leave name as is